app/src/components/integrations/containers/integrationInfoContainer/infoSection/infoSection.jsx

```diff
@@ -139,7 +141,12 @@ export class InfoSection extends Component {
                       })
                 }
               >
-                <InputSwitcher value={isEnabled} onChange={this.onToggleActiveHandler} />
+                <InputSwitcher
+                  value={isEnabled}
+                  onChange={() =>
+                    showPluginModal(isEnabled, title || name, this.toggleActiveHandler)
```

Comment:
I think will be better to edit `this.toggleActiveHandler` function to handle this case.